src/pages/public_registration/PublicPublicationInstance.tsx

```diff
@@ -43,7 +43,7 @@ const DisplayPages: FC<{ pages: PagesMonograph | null }> = ({ pages }) => {
 
   return pages?.pages ? (
     <LabelContentRow minimal label={`${t('common:details')}:`}>
-      `${t('references.pages')} ${pages.pages}`
+      {t('references.pages')} {pages.pages}
```

Comment:
```suggestion
      {t('references.pages')}: {pages.pages}
```
skal det ikke være et kolon der?

Reply:
sikkert mest fornuftig det ja! og i tillegg tror jeg det er feil oversettelse 😂 bør vel stå "Antall sider" og ikke "Sidetall" hehe

Reply:
finner mest feil på små PRer 😁

Reply:
haha ja, bortsett fra det var det feilfritt :D